### DISCUSSION TITLE
clang-format errors on commit prehooks

### DISCUSSION THREAD

**Discussion by Terentyev** [transmission/transmission#5037] [Support] [answered]
Hi, everybody!
I want to offer to community feature of old transmission versions: proxy settings for web-connections.
For some countries of our planet bittorrent trackers are blocking by ISP.

I implemented this functionality, but I couldn't commit my changes to my local created branch. In prehooks of commit clang-format output a lot of errors. One of this errorrs -- my changes. What should I do at this case?

[clang-formatted.log](https://github.com/transmission/transmission/files/10824065/clang-formatted.log)

**Comment by mikedld** [comment, marked as answer]
We're currently using clang-format version 15. If your locally installed version is different (looks like it's older), results may not be what our CI expects. If you do have v15 installed and still getting errors, you'll have to fix them, otherwise our CI will block the PR.

**Reply by Terentyev** [reply]
Thank you for your attention
